src/telegram_bot.py

```diff
@@ -0,0 +1,62 @@
+from os import getenv
+from telegram.ext import Updater, CommandHandler, CallbackContext
+import logging
+import schedule
+from requests import get
+from story import Story, get_new_stories
+
+# enables and get logger
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+					 level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# fetching env variables
+TOKEN = getenv("TELEGRAM_TOKEN")
+JSON_URL = getenv("JSON_URL")
+FETCH_INTERVAL = int(getenv("FETCH_INTERVAL", default=15)) # in minutes
+CHAT_ID = getenv("CHAT_ID")
+
+# save last story published
+last_story = None
+
+def start(update, context):
+	"""
+	Function handling the start of a conversation. it just send a welcome message
+	"""
+	update.message.reply_text("Ciao!")
+
+def error(update, context):
+	logger.warning('Update "%s" caused error "%s"', update, error)
+
+# creating updater and getting dispatcher
+updater = Updater(token = TOKEN, use_context = True) 
```

Comment:
stesso discorso per questi pezzi. In Python normalmente si mettono in una funzione chiamata `main` e li si richiama al lancio del programma